### PR TITLE
Screensharing

### PR DESCRIPTION
**Pull request by negzi**
Example for screen sharing.

### CODE REVIEW

**Review by stefanalund — CHANGES_REQUESTED** [review]
I think you should add a README.md file in the `screen-sharing` folder and describe how to create the extension from this "template". Include info on how to submit to Google Chrome Web Store.

File: screen-sharing/screenSharing.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        <div id="startbuttons">
+          <input value="Screen" onclick="startScreen();" type="button">
+        </div>
+        <video id="videoElement" style="height:100%; width:100%" autoplay="true" muted="true"></video>
+        <div style="display: none;" id="stopbuttons">
+          <input value="Stop" onclick="stopScreen();" type="button">
+        </div>
+</div>
+<script type="application/javascript">
+var video_status = false;
```
Comment (stefanalund) [inline screen-sharing/screenSharing.html]: This is not a very good variable name 😏

Comment (negzi) [reply screen-sharing/screenSharing.html]: hehe thats python :)

**Review by stefanalund — APPROVED** [review]
LGTM

**Review by stefanalund — APPROVED** [review]

**Pull request merged**